build/source/driver/multi_driver.f90

```diff
@@ -128,70 +143,78 @@ program multi_driver
-integer(i4b),parameter    :: ixRestart_im=1001              ! named variable to print a re-start file once per month
-integer(i4b),parameter    :: ixRestart_id=1002              ! named variable to print a re-start file once per day
-integer(i4b),parameter    :: ixRestart_never=1003           ! named variable to print a re-start file never
-integer(i4b)              :: ixRestart=ixRestart_im         ! define frequency to write restart files
+logical(lgt)                     :: printRestart               ! flag to print a re-start file
+integer(i4b),parameter           :: ixRestart_iy=1000          ! named variable to print a re-start file once per year
+integer(i4b),parameter           :: ixRestart_im=1001          ! named variable to print a re-start file once per month
+integer(i4b),parameter           :: ixRestart_id=1002          ! named variable to print a re-start file once per day
+integer(i4b),parameter           :: ixRestart_never=1003       ! named variable to print a re-start file never
+integer(i4b)                     :: ixRestart=ixRestart_im     ! define frequency to write restart files
 ! define output file
-character(len=8)          :: cdate1=''                      ! initial date
-character(len=10)         :: ctime1=''                      ! initial time
-character(len=64)         :: output_fileSuffix=''           ! suffix for the output file
-character(len=256)        :: summaFileManagerFile=''        ! path/name of file defining directories and files
-character(len=256)        :: fileout=''                     ! output filename
-! define model indices
-integer(i4b),allocatable  :: nSnow(:)                       ! number of snow layers for each HRU
-integer(i4b),allocatable  :: nSoil(:)                       ! number of soil layers for each HRU
-integer(i4b)              :: nLayers                        ! total number of layers
-real(dp),allocatable      :: dt_init(:)                     ! used to initialize the length of the sub-step for each HRU
-logical(lgt),allocatable  :: computeVegFlux(:)              ! flag to indicate if we are computing fluxes over vegetation (.false. means veg is buried with snow) 
+character(len=8)                 :: cdate1=''                  ! initial date
+character(len=10)                :: ctime1=''                  ! initial time
+character(len=64)                :: output_fileSuffix=''       ! suffix for the output file
+character(len=256)               :: summaFileManagerFile=''    ! path/name of file defining directories and files
+character(len=256)               :: fileout=''                 ! output filename
+! define model control structures
+integer(i4b)                     :: nLayers                    ! total number of layers
+integer(i4b),parameter           :: no=0                       ! .false.
+integer(i4b),parameter           :: yes=1                      ! .true.
```

Comment:
This was done in order to make use of the integer data structures (we do not currently have a logical data structure).